src/main/java/strategy/handlers/ReturnHandler.java

```diff
@@ -0,0 +1,12 @@
+package strategy.handlers;
+
+import db.Storage;
+import model.FruitTransaction;
+import strategy.OperationHandler;
+
+public class ReturnHandler implements OperationHandler {

```

Comment:
The class `ReturnHandler` should be in its own file, according to the Java convention where each top-level class is in a separate file. Ensure that the filename matches the class name.